File: src/test/java/de/gauss/aoc/ex2/Ex2Test.java
package de.gauss.aoc.ex2;

import de.gausss.aoc.ex2.Ex2;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class Ex2Test {
    private Ex2 exercise = new Ex2("ex2-test.txt");

    @Test
    void testA() {
        assertEquals(2, exercise.solveA());
    }

    @Test
    void testB() {
        assertEquals(1, exercise.solveB());
    }
}

File: src/test/kotlin/ex2/Ex2Test.kt
package ex2

import org.junit.jupiter.api.BeforeEach
import org.junit.jupiter.api.Test
import readInputLines
import kotlin.test.assertEquals


internal class Ex2Test {
    var input: List<String> = listOf();

    @BeforeEach
    fun setUp() {
        input = readInputLines("ex2-test.txt")
    }

    @Test
    fun testA() {
        assertEquals(2, ex2.solveA(input))

    }

    @Test
    fun testB() {
        assertEquals(1, ex2.solveB(input))

    }
}